src/main/java/cruise/umple/sample/downloader/ConsoleMain.java

```diff
@@ -145,7 +146,7 @@ public static void main(String[] args) throws IOException {
                 data.getInputContent().ifPresent(content -> {
                   data.getOutputPath().getParent().toFile().mkdir();
 
-                  EcoreImportHandler handler = new EcoreImportHandler();
+                  UmpleImportHandler handler = new EcoreImportHandler();
```

Comment:
This was just getting my code to compile with Warrens. But yeah, I was looking at what he did in his stuff, we need a factory to build `UmpleImportHandler`. I think I might bring [ImportType](https://github.com/umple-ucosp/umple.sample.downloader/blob/master/src/main/umple/ImportType.ump#L10) into the main project and add methods like: `getUmpleImporter()` that returns an instance of the proper importer, etc.